application/uitests/org.openjdk.jmc.test.jemmy/src/main/java/org/openjdk/jmc/test/jemmy/misc/wrappers/JfrWizard.java

```diff
@@ -446,6 +455,22 @@ private boolean setEventSetting(Boolean desiredState, String settingName, String
 		return currentState;
 	}
 
+	public void setEventSettingInvertOthers(boolean desiredState, String settingName, String ... path) {
+		// invert everything
+		// the actual names are observed to always at least contain the roots of the tree
+		// and setting them also sets all their children
+		{
+			final var eventSettingsTree = getEventSettingsTree();
+			final var names = eventSettingsTree.getAllItemTexts();
+			for (var name : names) {
+				final var currentPath = name.toArray(String[]::new);
+				setEventSetting(!desiredState, settingName, currentPath);
+			}
+		}
```

Comment:
Why are you using these additional block braces here?

Reply:
The block delimits the part that deals with inverting everything before setting the actual path to the desired value. I wanted to give it more structure than just writing it inline but I also felt it is overkill to extract that part into another method.

Reply:
OK, I personally don't like the block but again, matter of taste.